samples/AnomalyDetector/Generated/Models/AnomalyState.cs

```diff
@@ -31,7 +30,7 @@ internal AnomalyState(DateTimeOffset timestamp, AnomalyValue value, IReadOnlyLis
         {
             Timestamp = timestamp;
             Value = value;
-            Errors = errors.ToList();
```

Comment:
@AlexanderSher I believe this ToList had something to do with us owning the reference memory?  Can you help elaborate on the reasoning?

@ArcturusZhang if the ToList is needed we can change to
```c#
Errors = errors?.ToList() ?? new ChangeTrackingList<ErrorResponse>();
```

Reply:
No, this `ToList` comes from `System.Linq` therefore it did nothing.

We ensure the `ChangeTrackingList` in the deserialization method where we call this constructor using `Optional.ToList`, you could refer it here: https://github.com/Azure/autorest.csharp/blob/ab0215d968d7d35a47e3f3d429732974aaf3559d/samples/AnomalyDetector/Generated/Models/AnomalyState.Serialization.cs#L60
Therefore we do not need this `ToList` because we have already did the `ToList` and the parameter here will never get a null value.